### DISCUSSION TITLE
Reality Models

### DISCUSSION THREAD

**Discussion by hanan-commits-work** [iTwin/itwinjs-core#2966] [Q&A] [answered]
I am trying to query Reality Models in 3.0, previously in 2.1x we would simply use queryRealityData and then pass RealityDataQueryCriteria as a parameter, but I cannot find any replacement for it in 3.0, 

Anyone got any idea on how to accomplish this with 3.0?

**Comment by pmconne** [comment, marked as answer]
Use `RealityDataAccessClient` from [@itwin/reality-data-client](https://www.npmjs.com/package/@itwin/reality-data-client) package.
[Here's an example](https://github.com/iTwin/itwinjs-core/blob/master/test-apps/display-test-app/src/frontend/ClassificationsPanel.ts#L100) from display-test-app that populates a list of reality models intersecting a iModel's project extents.
